lib/udap_security_test_kit/registration_success_test.rb

```diff
@@ -60,7 +67,12 @@ class RegistrationSuccessTest < Inferno::Test
 
       post(udap_registration_endpoint, body: reg_body, headers: reg_headers)
 
-      assert_response_status(201)
+      if udap_client_registration_status == 'new'
+        assert_response_status(201)
+      elsif udap_client_registration_status == 'update'
+        assert_response_status(200)
```

Comment:
This is considerably more strict than the spec, which just says:

> If the Authorization Server returns the same client_id in the registration response for a modification request, it SHOULD also return a 200 OK HTTP response code.

Reply:
I think 201 or 200 for an update would be fine.

Reply:
Alright, I updated the code, unit tests, and descriptions to reflect this.